dspace-api/src/main/java/org/dspace/authenticate/IPAuthentication.java

```diff
@@ -176,7 +177,7 @@ public boolean isImplicit()
     {
         if (request == null)
         {
-            return ListUtils.EMPTY_LIST;
+            return Collections.EMPTY_LIST;
```

Comment:
In several places in the PR, it looks like you replace `ListUtils` with `Collections`, but leave the `import org.apache.commons.collections4.ListUtils` in place?  Is this on purpose, or should the `import` simply be removed here?